sidebarsDocs.js

```diff
@@ -139,8 +139,18 @@ module.exports = {
       link: { type: 'doc', id: 'network-iot/lorawan-network-servers/lorawan-network-servers' },
       items: [
         'network-iot/lorawan-network-servers/find-a-lns-provider',
-        'network-iot/lorawan-network-servers/run-a-lns',
-        'network-iot/buy-an-oui',
+        {
+          type: 'category',
+          label: 'Run Your Own LNS',
+          link: { type: 'doc', id: 'network-iot/run-an-lns/run-an-lns' },
+          items: [
```

Comment:
What I am missing here is the high level architecture of the helium network with your lns.

```
[dev] <=> [gateway] <=> [helium packet router] <=> [your lns] <=> [your integration]
                             ⇕                 ⇙
                        [iot config service]
```
With the explanation that your LNS only has to deal with the hpr and the iot config service. You won't get direct connections from the gateways; only from the hpr.